app/components/file/browse/display/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./file-list-display.component";
 export * from "./file-list-display-filter.pipe";
+export * from "./tree-view-display.component";
```

Comment:
maybe make a sub folder called tree-view where you have the component, scss, html and the helper methods